src/PIL/ImageFont.py

```diff
@@ -224,7 +228,7 @@ def __init__(
             raise core.ex
 
         if size <= 0:
-            msg = "font size must be greater than 0"
+            msg = f"font size must be greater than 0, not {size}"
```

Comment:
Was this also a source of confusion for you? I'm not sure why this wouldn't be suitably obvious.

Reply:
No this happened more on auto-pilot while coding in the evening -- I pretty much always try to include the incorrect value in error messages if it fails on input-validation as I find it makes debugging quite a lot easier for me (especially if the error is several frames down the stack from where I'm working).

Feel free to remove it, no hard feelings there :)